09-sliding_estimator.py

```diff
@@ -0,0 +1,82 @@
+"""
+=================
+Sliding estimator
+=================
+
+A sliding estimator fits a logistic legression model for every time point.
+In this example, we contrast the condition 'famous' against 'scrambled'
+using this approach. The end result is an averaging effect across sensors.
+The contrast across different sensors are combined into a single plot.
+
+Results script: :ref:`sphx_glr_auto_examples_statistics_plot_sliding_estimator.py`
+"""  # noqa: E501
+
+###############################################################################
+# Let us first import the libraries
+
+import os
+
+import numpy as np
+from scipy.io import savemat
+
+import mne
+from mne.decoding import SlidingEstimator, cross_val_multiscore
+
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegression
+
+from library.config import meg_dir, l_freq, N_JOBS, random_state
```

Comment:
`library` is not there.

I would do something like this, import the config and then add set the variables. Those that are defined in the config I would assign them here (not in the middle of the code) and those that are not in `config` but they are still relevant I would declare them here.

```py
import config

meg_dir = config.meg_dir
l_freq = config.l_freq
N_JOBS = config.n_jobs
random_state = config.random_state
DECIMATE = 4
```
constants should be CAPITAL